configuration.nix

```diff
@@ -30,4 +30,6 @@ in
   nix.extraOptions = ''
     experimental-features = nix-command flakes
   '';
+
+  system.stateVersion = "22.05";
```

Comment:
State version should normally not be changed when upgrading to a newer release from what I understand. It pins stuff like the postgres version so that the new version stays compatlible with the existing state. Setting it in configuration.nix makes sure that it does not get changed unless the user wants so